fs/proc/task_mmu.c

```diff
@@ -1182,7 +1181,6 @@ static int show_smaps_rollup(struct seq_file *m, void *v)
 
 	show_vma_header_prefix(m, priv->mm->mmap ? priv->mm->mmap->vm_start : 0,
 			       last_vma_end, 0, 0, 0, 0);
-	seq_pad(m, ' ');
```

Comment:
why is this necessary? removing line 1067 seems to be enough to fix it

Reply:
It's not necessary for the fix, but the extra space is also redundant since space is already added here.
https://github.com/ghostrider-reborn/android_kernel_common/blob/26f1f5d8e19d8e9f87ab952fed2668275889a480/fs/proc/task_mmu.c#L457